src/chrome/content/rules/Cyberport.hk.xml

```diff
@@ -0,0 +1,87 @@
+<!--
+	Non-functional hosts
+		Couldn't resolve host name:
+			 - conference.cyberport.hk
+			 - iciss.cyberport.hk
+			 - openstack4bday.cyberport.hk
+			 - registration.cyberport.hk
+			 - wine.cyberport.hk
+
+		Couldn't connect to server:
+			 - 2014gew.cyberport.hk
+			 - ac.cyberport.hk
+			 - angelhack.cyberport.hk
+			 - ct.cyberport.hk
+			 - cvcf.cyberport.hk
+			 - edm.cyberport.hk
+			 - hackathon.cyberport.hk
+			 - hackathon2014.cyberport.hk
+			 - istartup.cyberport.hk
+			 - license.cyberport.hk
+			 - ns1.cyberport.hk
+			 - tencent.cyberport.hk
+
+		Timeout was reached:
+			 - antispam1.cyberport.hk
+			 - cloud.cyberport.hk
+			 - extsmtp.cyberport.hk
+			 - ftp.cyberport.hk
+			 - gcloud.cyberport.hk
+			 - ictinternship.cyberport.hk
+			 - intra.cyberport.hk
+			 - intranet.cyberport.hk
+			 - ipv6.cyberport.hk
+			 - mail.cyberport.hk
+			 - mail2.cyberport.hk
+			 - map.cyberport.hk
+			 - mdmnsag.cyberport.hk
+			 - mdmstore.cyberport.hk
+			 - mdmxdm.cyberport.hk
+			 - osug.cyberport.hk
+			 - signup.cyberport.hk
+			 - tryipv6.cyberport.hk
+
+		SSL peer certificate or SSH remote key was not OK:
```

Comment:
This is a mistake because I have overloaded libcurl error in testing. Replaced `SSL peer certificate or SSH remote key was not OK` with `SSL peer certificate was not OK`. Thanks.